src/compressed_tensors/quantization/lifecycle/forward.py

```diff
@@ -356,12 +379,37 @@ def wrapped_forward(self, *args, **kwargs):
     setattr(module, "forward", bound_wrapped_forward)
 
 
-def maybe_calibrate_or_quantize(
+# TODO: to be moved out of compressed-tensors
```

Comment:
If we're aiming to get this refactor in before the next release, it'd be good practice o remove this note as well and replace with a warning or exception so we can note it and ensure it's caught -- mainly towards hygiene for the future and avoid the practice of adding TODOs (with the refactor with ruff one of the things is to check for TODOs and error on that for styling / linting)